### PR TITLE
Add rolling status to wormhole connections

### PR DESCRIPTION
**Pull request by itsionis**
The rolling status can be toggled like "save mass" and shows as a red badge on the connection with refresh style arrows as the icon.

### CODE REVIEW

File: js/app/init.js [hunk 1]
```diff
@@ -447,6 +447,17 @@ define([], () => {
...
                 ]
+            },  
+            wh_rolling: {
+                cssClass: 'pf-map-connection-preserve-mass',
+                overlays: [
+                    ['Label',
+                        {
+                            label: '<i class="fas fa-fw fa-sync-alt"></i>&nbsp;rolling',
+                            cssClass: ['pf-map-component-overlay', 'rolling'].join(' '),
+                            location: 0.35
```
Comment (itsionis) [inline js/app/init.js]: I suspect this value needs changing so it doesn't overlap with the save mass entry?

Comment (MonoliYoda) [reply js/app/init.js]: Have you tried setting both the Rolling and Save Mass flags?

Comment (itsionis) [reply js/app/init.js]: image

This is with the size indicator set to 0.5.

**Comment by MonoliYoda** [comment]
Merged onto dev branch

**Pull request closed without merging**